underfs/swift/src/main/java/tachyon/underfs/swift/direct/SwiftDirectOutputStream.java

```diff
@@ -0,0 +1,106 @@
+/*
```

Comment:
I would either remove "direct" from the package path or from the class name to avoid redundancy